content/fundamentals/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Introduction"
+title: "Fundamentals"
```

Comment:
It seemed odd to me to have an "Introduction" document in our "Introduction" section, so call this "Fundamentals" seemed like the right move.

Reply:
Ya, maybe we need another page later on that is an "introduction"

Reply:
Ah my comment might have made this confusing. This is the title to section -- So we still have our Introduction, but we're just considering Introduction, Philosophy, and Concepts as "Fundamentals" instead of "Introduction".